.github/workflows/cicd.yml

```diff
@@ -38,21 +40,16 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
-      - uses: subosito/flutter-action@v1
-      - run: flutter test --coverage --pub test
+      - uses: cedx/setup-dart@v2
+      - run: pub get
+      - run: dart pub run test
   
   benchmark:
     needs: [test]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v1
-      - uses: actions/setup-java@v1
-        with:
-          java-version: '12.x'
-      - uses: subosito/flutter-action@v1
-      - run: flutter pub get
-      - run: flutter pub run benchmark
+      - uses: cedx/setup-dart@v2
+      - run: pub get
+      - run: pub run benchmark
```

Comment:
Shouldn't be dart pub run here as well?

Reply:
The `pub` command is exposed on the PATH variable I believe. Wanted to test that, but having some issues with the yaml formatting :-)

Reply:
Ah I now got what you mean, in the test job I am running it with `dart pub run test`. I believe I can replace that with `dart run` to have the same effect. I'll update that

Reply:
Fixed and working.